packages/fiori/src/TimelineGroupItem.ts

```diff
@@ -6,7 +6,10 @@ import event from "@ui5/webcomponents-base/dist/decorators/event-strict.js";
 import jsxRenderer from "@ui5/webcomponents-base/dist/renderer/JsxRenderer.js";
 import TimelineLayout from "./types/TimelineLayout.js";
 import type { ITimelineItem } from "./Timeline.js";
-
+import "@ui5/webcomponents-icons/dist/slim-arrow-left.js";
```

Comment:
I don't see where you use these 4 icons, they are only imported, but no template uses them.

Is this a bug fix?

Icons are now imported in the template where they are used, not in the component file

Reply:
I added them, because without them in the Playground samples, when having `ui5-timeline-group-item`, the arrows in the buttons 
![image](https://github.com/user-attachments/assets/40b0de3d-07bf-4e25-acf8-dbefbee6d17b)
 weren't present, until imported explicitly.

Reply:
OK, I found the code:

```ts
get _groupItemIcon() {
		if (this.layout === TimelineLayout.Vertical) {
			return this.collapsed ? "slim-arrow-left" : "slim-arrow-down";
		}

		return this.collapsed ? "slim-arrow-up" : "slim-arrow-right";
	}
```

These icons were used, but were not imported, so it's a bugfix. After the TSX rework we recommend moving this to the template, and importing the icons from the template as named imports.

Please see `packages/main/src/MessageStripTemplate.tsx` for an example